### PR TITLE
Adding support for elements that have "style" attribute

### PR DESCRIPTION
**Pull request by tiagorg**
Hi @TimBeyer 

I am trying to use your library into https://github.com/tiagorg/Marionette.VDOMView where I am trying to implement VDOM on Marionette/Backbone Views.

Your library works great, however it doesn't support elements that have "style" attribute, I found that the VirtualDOM element needs to have styles as an object rather than a string, in order for the styles to be properly materialized (created as a real DOM element from the virtual DOM).

So I would like to propose this PR (which would also bump the version) so that mine would be able to start using yours :)

Any question please let me know. Thanks!

### CODE REVIEW

**Comment by review-ninja** [comment]
[ReviewNinja](http://app.review.ninja/TimBeyer/html-to-vdom/pull/8)

**Comment by TimBeyer** [comment]
Hi @tiagorg 

Thanks a lot for the PR!
The code looks fine. You just need to add some test cases for the style attribute parsing and then I can go ahead and merge it.  
Also, no need to bump the version in the PR. I'll take care of it when I do the release.

**Comment by tiagorg** [comment]
Absolutely! I've just added a test case for it which is expecting styles just like I need it in my lib.

I've also reverted the bumping.

Please let me know if this works.

File: test/html-to-vdom/index.js [hunk 1]
```diff
@@ -41,6 +41,19 @@ describe('htmlparser-to-vdom', function () {
             var converted = convertHTML(html);
             converted.properties.tabIndex.should.equal('1');
         });
+
+        it('parses an div with styles correctly', function () {
+
+            var html = '<div style="top: -7px; left: -6px; background: rgb(0,0,132);"></input>';
```
Comment (TimBeyer) [inline test/html-to-vdom/index.js]: Maybe some additional cases for missing spaces like `top:-7px;left:-6px;border:1px solid black;` might be useful.  
It also seems like browsers don't require a trailing semicolon eg: `style="top:-7px"` or `style="top: -7px; background:blue"`. I wonder if that should be supported as well.

**Comment by TimBeyer** [comment]
I left another comment. 
It would also be super awesome if you could `git rebase -i` your branch to remove the version bump change from history completely. That way everything's nice and clean.  

http://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history
http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by tiagorg** [comment]
Done and done. Please let me know if you see another deviation.

**Comment by TimBeyer** [comment]
Just one last thing and then we're good to go:
Your tests all open a `div` tag and then close an `input` tag. They should both be divs.

**Comment by tiagorg** [comment]
Ooops! Sorry, my mistake. Now it should be good :)

**Comment by TimBeyer** [comment]
Released as [v0.4.0](https://github.com/TimBeyer/html-to-vdom/releases/tag/v0.4.0) 
Thanks a lot for contributing!

**Comment by tiagorg** [comment]
Fantastic! Thanks very much Tim!

**Pull request merged**